src/clp_ffi_js/ir/StreamReader.cpp

```diff
@@ -46,31 +46,8 @@ auto StreamReader::create(DataArrayTsType const& data_array) -> StreamReader {
     auto zstd_decompressor{std::make_unique<clp::streaming_compression::zstd::Decompressor>()};
     zstd_decompressor->open(data_buffer.data(), length);
 
-    bool is_four_bytes_encoding{true};
```

Comment:
I had to remove this byte-encoding check as it seems to consume bytes from the reader and cause below `clp::ffi::ir_stream::Deserializer::create(*zstd_decompressor)` to fail with a protocol error. Do we still need this byte-encoding check?

I haven't taken a careful look into the clp-core code. Just out of curiosity, are we sticking with 4-byte or 8-byte encoding in IRfv2 or do we support both? I recall there were some discussions to deprecate one in favour of the other but I don't remember we had a conclusion.